### PR TITLE
OSSM-3732: Add istio postsubmit jobs

### PR DESCRIPTION
**Pull request by leoieggli**

### CODE REVIEW

File: ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml [hunk 1]
```diff
@@ -384,6 +384,21 @@ tests:
           memory: 2Gi
       timeout: 40m0s
     workflow: servicemesh-istio-e2e-aws
+- as: istio-update-operator-2.2
+  commands: |
+    ./maistra/scripts/run-test-infra-script.sh ./tools/automator.sh \
+    -o maistra \
+    -f /creds-github/github-token \
```
[the remark this answers is not in the document]
Comment (leoieggli) [reply ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml]: Is it not correct? https://github.com/maistra/istio/blob/maistra-2.4/maistra/scripts/run-test-infra-script.sh

**Comment by leoieggli** [comment]
/pj-reharse pull-ci-maistra-istio-maistra-2.3-istio-update-operator-2.3

**Comment by leoieggli** [comment]
/pj-reharse pull-ci-maistra-istio-maistra-2.4-istio-update-operator-2.4

**Comment by leoieggli** [comment]
/pj-rehearse pull-ci-maistra-istio-maistra-2.3-istio-update-operator-2.3

**Comment by leoieggli** [comment]
/pj-rehearse pull-ci-maistra-istio-maistra-2.4-istio-update-operator-2.4

File: ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml [hunk 2]
```diff
@@ -384,6 +384,20 @@ tests:
           memory: 2Gi
       timeout: 40m0s
     workflow: servicemesh-istio-e2e-aws
+- as: istio-update-operator-2.2
+  commands: |
+    ./maistra/scripts/run-test-infra-script.sh ./tools/automator.sh \
+    -o maistra \
+    -f /creds-github/token \
+    -r istio-operator \
+    -c make clean update-2.2-charts \
```
Comment (jwendell) [inline ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml:393]: Quote this line:

`'-c make clean update-2.2-charts'`

Like the line below. This is because it contains spaces.

also in 2.3 and 2.4 jobs below.

**Review by jwendell — CHANGES_REQUESTED** [review]

**Comment by leoieggli** [comment]
/pj-rehearse pull-ci-maistra-istio-maistra-2.4-istio-update-operator-2.4

**Comment by leoieggli** [comment]
/pj-rehearse pull-ci-maistra-istio-maistra-2.3-istio-update-operator-2.3

**Comment by leoieggli** [comment]
/pj-rehearse pull-ci-maistra-istio-maistra-2.4-istio-update-operator-2.4

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -250,13 +250,15 @@ aliases:
   - dgn
   - bartoszmajsak
   - FilipB
+  - leoieggli
```
Comment (jwendell) [inline OWNERS_ALIASES:253]: please revert this. Make a separate PR for this purpose.

**Review by jwendell — APPROVED** [review]

File: ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml [hunk 4]
```diff
@@ -384,6 +384,20 @@ tests:
...
     workflow: servicemesh-istio-e2e-aws
+- as: istio-update-operator-2.2
+  commands: |
+    ./maistra/scripts/run-test-infra-script.sh ./tools/automator.sh \
+    -o maistra \
+    -f /creds-github/token \
+    -r istio-operator \
+    '-c make clean update-2.2-charts' \
+    '-t Automator: Update charts from maistra/istio' \
+    -m update_charts_istio
```
Comment (jwendell) [inline ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml:395]: You probably missed the branch `-b` argument:

`-b maistra-2.2` , `-b maistra-2.3` etc

**Review by jwendell — CHANGES_REQUESTED** [review]

**Comment by leoieggli** [comment]
/pj-rehearse pull-ci-maistra-istio-maistra-2.3-istio-update-operator-2.3 pull-ci-maistra-istio-maistra-2.4-istio-update-operator-2.4
